extensions/amp-analytics/0.1/transport.js

```diff
@@ -118,13 +152,215 @@ export class Transport {
     xhr.send('');
     return true;
   }
+
+  /**
+   * If iframe is specified in config/transport, check whether third-party
+   * iframe already exists, and if not, create it.
+   * @param {!Window} win The window element
+   * @param {!Object<string,string>} transportOptions The 'transport' portion
+   * of the amp-analytics config object
+   * @param {function(!string,
+   *   !../../../src/3p-analytics-common.AmpAnalytics3pResponse)=}
+   *   opt_processResponse An optional function to receive any response
+   *   messages back from the cross-domain iframe
+   */
+  processCrossDomainIframe(win, transportOptions, opt_processResponse) {
+    const frameUrl = dev().assertString(transportOptions['iframe'],
+      'Cross-domain frame parameters missing ${this.type_}');
+    const frameData = this.useExistingOrCreateCrossDomainIframe_(
+      win, frameUrl, transportOptions['extraData']);
+    const iframeMessagingClient = frameData.iframeMessagingClient;
+    iframeMessagingClient.registerCallback(
+      AMP_ANALYTICS_3P_MESSAGE_TYPE.READY, () => {
+        iframeMessagingClient.setHostWindow(frameData.frame.contentWindow);
+        frameData.newCreativeMessageQueue.setIsReady();
+        frameData.eventQueue.setIsReady();
+      });
+    iframeMessagingClient.registerCallback(
+      AMP_ANALYTICS_3P_MESSAGE_TYPE.RESPONSE,
+        response => {
+          dev().assert(response &&
+            response[AMP_ANALYTICS_3P_MESSAGE_TYPE.RESPONSE],
+            'Received empty response from 3p analytics frame');
+          if (!opt_processResponse) {
+            dev().warn(TAG_, 'Received response from 3p analytics frame when' +
+              ' none was expected');
+          }
+          opt_processResponse(
+            this.type_,
+            /** @type {!../../../src/3p-analytics-common.AmpAnalytics3pResponse} */ (response)); // eslint-disable-line max-len
```

Comment:
Can you just put (response)) on next line so you can remove the disable line length?